apache/security.conf

```diff
@@ -0,0 +1,3 @@
+ServerName localhost
```

Comment:
Migrated this from config previously inline in the Dockerfile. Not sure if it matters with signature off but seems simpler to keep it than to dig into exactly how we ended up with it before.